test/primer/view_helper_test.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class Primer::ViewHelperTest < Minitest::Test
+  include Primer::ViewHelper
+  include Primer::ComponentTestHelpers
+
+  # The helper calls #render, but it is not available in tests
+  alias render render_inline
```

Comment:
Do we have an integration test that uses the full Rails stack that we could utilize for these tests?

I don't think it's worth adding one in this iteration if we don't, but it could be beneficial if the tests went through the full rendering pipeline instead of through `render_inline`.

Reply:
we don't right now, but we are working on it!